otter/convergence/model.py

```diff
@@ -572,11 +573,20 @@ class CLB(object):
     health_monitor = attr.ib(default=False)
 
 
+@attr.s
+class DrainingUnavailable(Exception):
+    """
+    Exception raised when draining info is required but is not available
+    """
+    lb_id = attr.ib()
+    node_id = attr.ib()
+
+
 @implementer(ILBNode, IDrainable)
 @attributes([Attribute("node_id", instance_of=basestring),
              Attribute("description", instance_of=CLBDescription),
              Attribute("address", instance_of=basestring),
-             Attribute("drained_at", default_value=0.0, instance_of=float),
+             Attribute("drained_at", default_value=None),
```

Comment:
Active nothing! Adding `None` to `drained_at` means needing to check every single access point in the codebase.  Perhaps this should be a private `attr` with a public `@property` that raises `DrainingUnavailable`?